### DISCUSSION TITLE
2024-10-02(Web) Online mentoring 6 - hold

### DISCUSSION THREAD

**Discussion by jongfeel** [tttghost/wheelmaker#52] [General]
일정상으로는 내일 모레 10월 2일 수요일이 멘토링 일정인데
회사에서 하는 일이 바쁘고, 이사도 하셔야 하고 해서 잠정적으로 쉬는 기간을 가지고 가면 좋겠습니다.
이 논의는 지난 주 책 전달 드리고 점심 같이 하면서 한 얘기긴 합니다.

그래서 다음에 연락 주실 때 결정할 내용은

- 언제쯤 멘토링을 다시 재개할 수 있는지, 10월 중으로 결정되면 좋음 (best)
- 기존 wheelmaker에 대한 개발 진행을 어느 부분 까지 하고 완료하면 좋을지 범위 결정
- 개발 진행하면서 singleton 구조에 대한 코드 리뷰 진행, 꼭 했으면 좋겠습니다.

10월을 넘기지 않는 시점에 재개 했으면 하고
잊을 때 쯤 제가 리마인드 연락은 드려 보겠습니다.

**Comment by tttghost** [comment]
네, 종필님. 프로젝트 마무리기간이 핵심기능 개발이 10월말 정도가 될 것 같습니다.
10월 23일에 다시 시작하는 것은 어떨까요??
싱글톤 구조 코드리뷰 진짜진짜.. 마음만큼은 굴뚝같은데 아직 제 노오력이 부족한 것 같습니다.

10월 23일에 다시 시작하겠습니다..!

**Comment by jongfeel** [comment]
개인 사정으로 인해 일정 연기
10월 23일 시점에 연기한 것으로
프로젝트 진행이나 코드 리뷰 준비가 안되어 있는 것으로 판단

프로젝트가 아니어도 좋으니
다시 뭔가 얘기하고 싶은 시점이 오면 얘기해 주세요.
그때가 다시 시작할 떄입니다.

제 바램은 올해를 넘기지 않았으면 합니다.

**Comment by jongfeel** [comment]
2024-11-06 부터 진행하기로 결정
